RealEstate/3 reports.sql

```diff
@@ -1,21 +1,21 @@
 --1) I want a report of all the houses sorted by the block and then by the town/city
-select *
+select [Address], Town, Realtor, [Owner], DateSold, Buyer 
```

Comment:
I don't want to see any nulls in my reports, please use the ISNULL function.

Reply:
For `varchar` fields, `null` is not allowed in the table definition so it's not an issue. For `date` fields, there's no "blank" value to put into the `isnull` function. A `null` date will generally render as a blank date control in front ends. It is possible to convert to `varchar` and render `null` as blank, but then you lose any sorting/operations that can be done on date fields. Is that what you'd prefer?